raiden/tests/integration/test_raidenservice.py

```diff
@@ -348,3 +357,59 @@ def get_channel_state(app) -> NettingChannelState:
     ]
 
     assert channel_state.fee_schedule.imbalance_penalty == full_imbalance_penalty
+
+
+@raise_on_failure
+@pytest.mark.parametrize("number_of_nodes", [2])
+@pytest.mark.parametrize("channels_per_node", [0])
+@pytest.mark.parametrize("number_of_tokens", [1])
+def test_blockchain_event_processed_interleaved(
```

Comment:
When adding failing tests, you could add `@pytest.mark.xfail` to make that even clearer. It will also help if someone tries to bisect the commits.

Reply:
👍